packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -299,7 +299,7 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     }
 
     return this.teeErrorToEventEmitter(() => {
-      const audioOptions: AudioOptions = { muted: microphoneOn ?? !this.getState().isLocalPreviewMicrophoneEnabled };
+      const audioOptions: AudioOptions = { muted: !microphoneOn ?? !this.getState().isLocalPreviewMicrophoneEnabled };
```

Comment:
`microphoneOn` is optional. If we check for falsiness, calling `joinCall()` without any arguments will cause `muted` to be set to `true`.

So, fallback on `undefined`, and *then* check for falsiness...

```suggestion
      const audioOptions: AudioOptions = { muted: !(microphoneOn ?? this.getState().isLocalPreviewMicrophoneEnabled) };
```

Reply:
Great suggestion. I've fixed my code.